guest/rust/examples/assets/unity/download.sh

```diff
@@ -0,0 +1,37 @@
+export BASE="${EXAMPLES_ASSETS_HOST}/unity/Dynamic%20Nature%20-%20Mountain%20Tree%20Pack"
```

Comment:
Is EXAMPLES_ASSETS_HOST defined somewhere?

Reply:
Yeah it's a github secret, because it's not public content

Reply:
(In the future we should switch to public content, but to get started I wanted to use something I know at least works somewhat)